css/index.css

```
@@ -1,3 +1,296 @@
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: space-around;
+  margin: 50px 0 50px 0;
+}
+.cta {
+  text-align: center; 
+  line-height: 60px;
+}
+
+.cta h1 {
+	font-size: 72px;
+}
+
+.about-services {
+  margin: 30px 0 0 30px;
+}
+
+.about-services h1 {
+	font-size: 24px;  
+}
+
+button {
+  font-size: 16px;
+  padding: 5px 40px 5px 40px;
+  border: 1px solid black;
+}
+
+.content {
+  border-top: 2px solid black;
+  border-bottom: 2px solid black;  
+  margin: 30px; 
+}
+
+.services-content {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;  
+  border-top: 1px solid black;
+  border-bottom: 1px solid black;  
+  margin: 30px;
+  padding-bottom: 40px;
+}
+
+.service-item {
+  margin: 40px 0 0 0;
+  padding: 20px;
+  background: lightgrey;
+  border: 1px solid black;
+  width: 400px;
+}
+
+.service-item p {
+  line-height: 18px;
+}
+.service-item button {
+  margin-top: 20px;
+  font-size: 12px;
+  padding: 7px 20px 7px 20px;
+  border: 1px solid black;  
+  border-radius: 5px;
+}
+
+.two-col {
+  display: flex;
+  justify-content: space-between;
+  margin: 40px 0 40px 0;
+}
+
+.upper-items {
+ width: 400px;
+}
+
+.three-col {
+  display: flex;
+  justify-content: space-between;  
+  margin: 40px 0 40px 0;
+}
+
+.lower-items {
+  width: 250px;
+}
+
+footer {
+  margin: 20px 30px 90px 30px;
+}
+
+.copyright {
+  text-align: center;
+  font-size: 14px;
+  margin-top: 50px;
+}
+
+address {
+  margin-bottom: 20px;
+}
+
+.service-facts {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;   
+  margin: 30px;
+}
+
+.service-facts-item {
+  width: 400px;
+}
+
+@media (max-width: 800px) {
+    .container {
+        width: 800px;
+    }
+    header nav a {
+        margin: 44px;
+    }
+    img.logo {
+        margin: 20px 20px 0 20px;
+        padding-top: 20px;
+    }
+    .hero {
+        justify-content: center;
+        margin: 30px 0 50px 0;
+    }
+    img.cta {
+        display: none;
+    }
+    .upper-items {
+        width: 360px;
+    }
+    .lower-items {
+        width: 210px;
+    }
+}
+
+@media (max-width: 500px) {
+    .container {
+        width: 500px;
+    }
+    header {
+        flex-direction: column-reverse;
+        justify-content: center;
+    }
+    header nav {
+        display: flex;
+        flex-flow: column nowrap;
+        border-top: 1px solid grey;
+        margin: 0 50px 0 50px;
+    }
+    header nav a {
+        text-align: center;
+        border-bottom: 1px solid grey;
+        padding: 20px;
+        margin: 0;
+    }
+    img.logo {
+        width: 183px;
+        height: 91px;
+        margin: 0 auto;
+        padding: 35px 0 20px 0;
+    }
+    .hero {
+        justify-content: center;
+        margin: 60px 0 50px 0;
+    }
+    img.cta {
+        display: none;
+    }
+    .two-col {
+        display: flex;
+        flex-flow: column nowrap;
+        margin: 40px 0 0 0;
+    }
+    .upper-items {
+        margin-bottom: 50px;
```

```suggestion
        margin-bottom: 50px;
        width: 100%;
```
If you follow my suggestions for this class, add this width property for this breakpoint